app.go

```diff
@@ -85,3 +88,11 @@ func Automatic(opts Options) *App {
 
 	return a
 }
+
+func (a *App) addDefaultRouteNames() {
```

Comment:
why would we need this? shouldn't all routes get names when they're first added?